dev/build/windows/makecoq_mingw.sh

```diff
@@ -1153,9 +1153,13 @@ function make_lablgtk {
   make_gtk_sourceview3
   make_ocaml_cairo2
 
-  if build_prep https://github.com/garrigue/lablgtk/archive  3.0.beta5  tar.gz 1 lablgtk-3.0.beta5 ; then
+  if build_prep https://github.com/garrigue/lablgtk/archive  3.1.1  tar.gz 1 lablgtk-3.1.1 ; then
     make_arch_pkg_config
 
+    # We should download the artifact, not the tag, since the tag has no
+    # version number. We remedy
```

Comment:
What's the artifact and why do we not download it?

Reply:
If you look at https://github.com/garrigue/lablgtk/releases/tag/3.1.1, you will notice that there is both a tbz file and a tgz file for the release. The tbz file is the artifact, while the tgz file is the tag. Contrarily to the general intuition (at least mine), the content of the archived files are actually different (that is a feature of dune-release). No idea why we are not downloading the artifact; presumably because `build_prep` cannot locate it (notice how different their URLs are).

Reply:
Dune is making hard things simple, and simple things hard :-)